### PR TITLE
Revised branch

### PR DESCRIPTION
**Pull request by rtingchaleun**
fixed branch errors and review comments

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,30 +24,42 @@
       </div>
 
       <div class="form-container center">
```
Comment (bolah2009) [inline index.html]: `OPTIONAL` 

Consider using `main` instead, this shows the dominant content of the `<body>` of a document. More on `<main>` [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) :star2:

File: README.md [hunk 2]
```diff
@@ -1,2 +1,11 @@
...
+Embedding-images-and-Video-Microverse
+
+# Author
+Roberto Erik Martinez and Ronald Tingchaleun
+
+# Project Website
+https://www.theodinproject.com/courses/html5-and-css3/lessons/html-forms
+
+# Project Description
+Learning how to code HTML forms
```
Comment (bolah2009) [inline README.md]: `AWESOME`

Nice job creating a `README.md` file, this gives the project more description of what it does and how it does it.

-----

`REQUIRED`

> Remember about descriptive README file.`

- Add the assignment link, the project specification, link to clone page and source page, screenshots and other pieces of information that would describe the project.

File: index.html [hunk 3]
```diff
@@ -24,30 +24,42 @@
...
+        <div class="padding-25">
+          <h1>Create an Intuit account</h1>
+          <p>
+            One account for everything Intuit, including Mint.<br>
+            <a href="https://accounts-help.lc.intuit.com/questions/1582580-creating-an-account">Learn more</a>
+          </p>
+          <form>
+            <div class="form-field">
+              <label for="input-email">Email Address</label>
+              <input id="input-email" type="email" name="email">
```
Comment (bolah2009) [inline index.html]: `AWESOME`

Good job associating the `<label>` with an `<input>` element. :clap: :+1: 

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```


You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)

File: index.html [hunk 4]
```diff
@@ -24,30 +24,42 @@
       </div>
```
Comment (bolah2009) [inline index.html]: `REQUIRED`

The header image and icons seem to be missing.

| Source Page |
|:-----:|
|  html form icons source |

**Comment by Oghenebrume50** [comment]
Reviewing....

**Pull request merged**